packages/ui/src/ui/components/Dialog/controls/RoleListControl/utils.ts

```diff
@@ -0,0 +1,108 @@
+import cn from 'bem-cn-lite';
+
+import filter_ from 'lodash/filter';
+import forEach_ from 'lodash/forEach';
+import map_ from 'lodash/map';
+
+import type {EditableManyListsItemType} from '../../../../components/Dialog';
+import type {ResponsibleType, RoleConverted} from '../../../../utils/acl/acl-types';
+import type {PreparedRole} from '../../../../utils/acl';
+import {RoleListControlProps} from './RoleListControl';
+
+export const block = cn('role-list-control');
+
+export function prepareRoleListValue(roles: Array<PreparedRole>, otherMembers: Array<string> = []) {
```

Comment:
I would recommend not putting all the features in one bucket. But still distribute them to different files. They are then a little easier to search and test. You won't need to pull the entire module

Reply:
Storing helpers in a single file is justified if they are linked. Separate storage allows you to avoid name conflicts and optimize your build